src/Component/src/Metadata/Resource/Factory/AttributesResourceMetadataCollectionFactory.php

```diff
@@ -143,20 +149,6 @@ private function getOperationWithDefaults(ResourceMetadata $resource, Operation
 
         $operation = $operation->withResource($resource);
 
-        if (null === $resource->getName()) {
```

Comment:
A question about line of code above this code.

```php
        $resourceConfiguration = $this->resourceRegistry->get($resource->getAlias() ?? '');
```

We probably should assert it as `Assert::notNull($resource->getAlias());` as was in some other PR/code.